### PR TITLE
add kliff

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
Only the license is missing - I contacted the original developer and wait for their reply.

**Comment by mbargull** [comment]
It's probably okay to use https://github.com/mjwen/kliff/blob/57a5a4bee0708640325b1aeeea830d6a168339df/LICENSE . They didn't merge that commit into their `master` branch yet, but https://github.com/mjwen/kliff/blob/v0.1.4/setup.py#L111 states it is `CDDL-1.0`.
So, I'd go for
```yaml
  license: CDDL-1.0
  license_file: LICENSE
```
and add the `LICENSE` from the above commit.
Thanks for raising the issue upstream!

File: recipes/kliff/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  host:
+    - python
+    - pip
+    - pybind11
+  run:
+    - python
+    - scipy
+    - pybind11
+    - pytest
+    - kim-api
```
Comment (mbargull) [inline recipes/kliff/meta.yaml]: https://github.com/mjwen/kliff/blob/v0.1.4/docs/source/installation.rst mentions `kimpy`. Is this something we need here?

File: recipes/kliff/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5df8088517cd45f6bb893ced3a483a84ccb33f89815f6e7f3b55c0478289b91f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```
Comment (mbargull) [inline recipes/kliff/meta.yaml]:
```suggestion
  skip: true  # [win or py<35]
```

File: recipes/kliff/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipes/kliff/meta.yaml]: Yes, it should be `kimpy` rather than the `kim-api` package.

**Comment by mbargull** [comment]
Needs a merge with `conda-forge:master` due to us merging gh-10228 and then it should be good to go, I think.

**Comment by jan-janssen** [comment]
@mbargull Thanks for your review, I guess it is now ready to be merged.

**Pull request merged**